common/static/common/templates/discussion/customwmd-prompt.underscore

```diff
@@ -0,0 +1,44 @@
+<h4 id="editor-dialog-title"><%= title %></h4>
+<div role="status" id="wmd-editor-dialog-form-errors" class="sr" tabindex="-1"></div>
+<div>
+    <form class="form">
+        <fieldset class="field-group">
+            <legend class="form-group-hd sr"><%= title %></legend>
+            <div class="field <%= uploadFieldClass %>">
```

Comment:
@andy-armstrong I think that works for all cases in this file except for [one of the descriptions](https://github.com/edx/edx-platform/blob/clytwynec/ac-75/common/static/common/templates/discussion/customwmd-prompt.underscore#L30), which [has a link in it in](https://github.com/edx/edx-platform/blob/clytwynec/ac-75/lms/static/js/Markdown.Editor.js#L42-L45).  In this particular file, we aren't loading any user provided data (only the stuff already defined in our JS).  Does it make sense to use `<%=` in this case?